fastai/data/core.py

```diff
@@ -323,7 +323,14 @@ def show_at(o, idx, **kwargs):
 @delegates(TfmdLists)
 class Datasets(FilteredBase):
     "A dataset that creates a tuple from each `tfms`"
-    def __init__(self, items=None, tfms=None, tls=None, n_inp=None, dl_type=None, **kwargs):
+    def __init__(self,
+        items:list=None, # List of items to create `Datasets`
+        tfms:(list,Pipeline)=None, # List of `Transform`(s) or `Pipeline` to apply
+        tls:TfmdLists=None, # If None, `self.tls` is generated from `items` and `tfms`
+        n_inp:int=None, # Number of elements in `Datasets` tuple that should be considered part of input
+        dl_type=None, # Type of `DataLoader`
```

Comment:
```suggestion
        dl_type=None, # Default type of `DataLoader` used when function `FilteredBase.dataloaders` is called. 
```
This kind of gives more of an idea on what dl_type affects.